gateway/test/src/test/resources/test-plan-data/simple/simple-connector-interceptor-policy/connector-interceptor/002-intercepted-response.resttest

```diff
@@ -0,0 +1,13 @@
+PUT /gateway/SimpleConnectorInterceptorPolicyTest/connector-interceptor-policy/1.0/path/to/app/resource admin/admin
+Content-Type: application/json
+
+{
+  "intercept" : "true"
```

Comment:
Your test policy is looking for "intercept" in the request headers, not in the body of the request.  So I think this needs to be:

```
Content-Type: application/json
intercept: true
```